components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -661,25 +676,28 @@
                             %>
                         </td>
                         <td class="sectionHelp" width="50%" style="text-align:left; !important">
-                            <%=description%>
+                            <%=Encode.forHtml(description)%>
                         </td>
 
                     </tr>
                     <%
                                 }else{
                                     %>
-                                   <td class="leftCol-med" width="50%" style="display:none;" id="<%=name%>"><%=propertyName%>
+                                   <td class="leftCol-med" width="50%" style="display:none;" id="<%=Encode.forHtmlAttribute(name)%>"><%=Encode.forHtml(propertyName)%>
```

Comment:
variable name is hardcoded and does not contain special characters. No need to encode that variable